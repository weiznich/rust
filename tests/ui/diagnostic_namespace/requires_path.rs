#![feature(diagnostic_namespace)]
// check-fail

#[diagnostic]
//~^ERROR Diagnostic attributes requires a path with 2 segments
//~|ERROR Diagnostic attributes requires a path with 2 segments
//~|ERROR cannot find attribute `diagnostic` in this scope
pub struct Bar;


fn main() {
}
