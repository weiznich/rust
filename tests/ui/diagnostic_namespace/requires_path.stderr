error: Diagnostic attributes requires a path with 2 segments
  --> $DIR/requires_path.rs:4:3
   |
LL | #[diagnostic]
   |   ^^^^^^^^^^

error: Diagnostic attributes requires a path with 2 segments
  --> $DIR/requires_path.rs:4:3
   |
LL | #[diagnostic]
   |   ^^^^^^^^^^

error: cannot find attribute `diagnostic` in this scope
  --> $DIR/requires_path.rs:4:3
   |
LL | #[diagnostic]
   |   ^^^^^^^^^^

error: aborting due to 3 previous errors

